config/validator.py

```diff
@@ -1,5 +1,6 @@
 ## Format of return will be [Happy: bool, error: str]
 from os import path
+from email.utils import parseaddr, formataddr
```

Comment:
Is this a core dep, or does it require an additional package?

Reply:
It's part of the Python standard library. It's provided by libpython3-stdlib in Debian, but this is a hard-dep of the python3 package. Even if we were to use python3-minimal, we would still get email.utils as part of libpython3.11-minimal